tests/scripts/all.sh

```diff
@@ -403,7 +423,19 @@ scripts/config.pl unset MBEDTLS_THREADING_C
 scripts/config.pl unset MBEDTLS_MEMORY_BACKTRACE # execinfo.h
 scripts/config.pl unset MBEDTLS_MEMORY_BUFFER_ALLOC_C # calls exit
 scripts/config.pl unset MBEDTLS_PLATFORM_TIME_ALT # depends on MBEDTLS_HAVE_TIME
-CC=armcc AR=armar WARNING_CFLAGS= make lib
+
+CC="$ARMC5_CC" AR="$ARMC5_AR" WARNING_CFLAGS= make lib
```

Comment:
I think the CFLAGS below should be tighter. Fail on warnings, pedantic, strict etc.